packages/suite-native-atoms/src/Button/TileButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { Icon, IconName } from '@trezor/icons';
+
+type ActionIconButtonProps = {
+    iconName: IconName;
+    title: string;
+    onPress: () => void;
+    style?: NativeStyleObject;
+};
+
+const rectangleButtonStyle = prepareNativeStyle(utils => ({
+    height: 79,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: utils.borders.radii.small,
+    backgroundColor: utils.colors.gray300,
+    flex: 1,
+}));
+
+export const TileButton = ({ iconName, title, onPress, style }: ActionIconButtonProps) => {
```

Comment:
It will also be used in actions modal (send / receive) so it will be shared across 2 pages. The way I think about it is if I use it in one page, don't add it to atoms but if it's used in more than 1 page, then put it to atoms for reusable cmp.